src/test/java/com/woowacamp/soolsool/core/member/service/MemberServiceTest.java

```diff
@@ -182,11 +180,61 @@ void removeMember() {
 
         // when
         when(memberRepository.findById(userId)).thenReturn(Optional.of(member));
-        Assertions.assertThatNoException()
+        assertThatNoException()
             .isThrownBy(() -> memberService.removeMember(userId));
 
         // then
         verify(memberRepository).findById(userId);
         verify(memberRepository).delete(member);
     }
+
+    @Test
+    @DisplayName("성공 : 마일리지 충전")
+    void chargeMemberMileage() {
+        // given
+        String amount = "10000";
+        MemberMileageChargeRequest memberMileageChargeRequest = new MemberMileageChargeRequest(
+            amount
+        );
+
+        Member member = Member.builder()
+            .role(MemberRole.builder()
+                .name(MemberRoleType.CUSTOMER)
+                .build())
+            .email("test@email.com")
+            .password("test_password")
+            .name("최배달")
+            .phoneNumber("010-1234-5678")
+            .mileage("5000")
+            .address("서울시 잠실역")
+            .build();
+
+        MemberMileageCharge memberMileageCharge = MemberMileageCharge.builder()
+            .member(member)
+            .amount(amount)
+            .build();
+
+        // when
+        when(memberRepository.findById(anyLong())).thenReturn(Optional.of(member));
+        when(memberRepository.save(any(Member.class))).thenReturn(member);
+        when(memberMileageChargeRepository.save(any(MemberMileageCharge.class)))
+            .thenReturn(memberMileageCharge);
+
+        memberService.addMemberMileage(1L, memberMileageChargeRequest);
+
+        // then
+        verify(memberRepository).findById(anyLong());
+        ArgumentCaptor<Member> memberArgumentCaptor = ArgumentCaptor.forClass(Member.class);
+        verify(memberRepository).save(memberArgumentCaptor.capture());
+        ArgumentCaptor<MemberMileageCharge> memberMileageChargeArgumentCaptor = ArgumentCaptor
+            .forClass(MemberMileageCharge.class);
+        verify(memberMileageChargeRepository).save(memberMileageChargeArgumentCaptor.capture());
+
+        assertAll(
+            () -> assertThat(memberArgumentCaptor.getValue().getMileage().getMileage())
+                .isEqualTo(new BigInteger("15000")),
+            () -> assertThat(memberMileageChargeArgumentCaptor.getValue().getAmount().getMileage())
+                .isEqualTo(new BigInteger(amount))
```

Comment:
```suggestion
        // then
        ArgumentCaptor<Member> memberArgumentCaptor = ArgumentCaptor.forClass(Member.class);
        ArgumentCaptor<MemberMileageCharge> memberMileageChargeArgumentCaptor =
            ArgumentCaptor.forClass(MemberMileageCharge.class);
        
        verify(memberRepository).findById(anyLong());
        verify(memberRepository).save(memberArgumentCaptor.capture());
        verify(memberMileageChargeRepository).save(memberMileageChargeArgumentCaptor.capture());

        assertAll(
            () -> assertThat(memberArgumentCaptor.getValue().getMileage().getMileage())
                .isEqualTo(new BigInteger("15000")),
            () -> assertThat(memberMileageChargeArgumentCaptor.getValue().getAmount().getMileage())
                .isEqualTo(new BigInteger(amount))
        );
```

저는 이게 더 가독성이 좋은것 같아요.

Reply:
굳~~